packages/okta-react-native/package.json

```diff
@@ -1,59 +1,44 @@
 {
   "name": "@okta/okta-react-native",
-  "version": "0.1.2",
-  "description": "React Native support for OpenID Connect",
-  "main": "src/token-client.js",
+  "title": "React Native Okta Sdk Bridge",
+  "version": "1.0.0",
+  "description": "Okta OIDC for React Native",
+  "main": "index.js",
   "scripts": {
-    "test": "yarn test:unit",
-    "test:unit": "jest --coverage --verbose --testPathPattern __tests__/.*.test.js"
-  },
-  "jest": {
-    "browser": true,
-    "preset": "jest-expo",
-    "automock": false,
-    "setupFiles": [
-      "./setupJest.js"
-    ],
-    "transformIgnorePatterns": [
-      "node_modules/(?!(jest-)?react-native|react-clone-referenced-element|@react-native-community|expo(nent)?|react-navigation|@react-navigation/.*|@unimodules/.*|sentry-expo|native-base)"
-    ]
+    "test": "echo \"Error: no test specified\"",
```

Comment:
Do we really have no way to run tests?

Reply:
Tests isn't a part of this PR. They will be introduced later in the next few weeks!

Reply:
👍